src/store/participantsStore.js

```diff
@@ -600,12 +617,27 @@ const actions = {
 
 		try {
 			const response = await request(token)
-			context.dispatch('purgeParticipantsStore', token)
-
 			const hasUserStatuses = !!response.headers['x-nextcloud-has-user-statuses']
 
-			response.data.ocs.data.forEach(participant => {
-				context.dispatch('addParticipant', { token, participant })
+			const currentParticipants = context.state.attendees[token]
+			const newParticipants = response.data.ocs.data
+			for (const attendeeId of Object.keys(Object(currentParticipants))) {
```

Comment:
Object wrapping is not necessary

Reply:
`currentParticipants` can be undefined at this moment.
So either this [wrapping](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/Object#return_value) or `currentParticipants !== undefined`